proto/mezo/bridge/v1/bridge.proto

```diff
@@ -40,10 +40,33 @@ message AssetsLockedEvent {
   string token = 4;
 }
 
-// ERC20TokenMapping defines a mapping between an ERC20 token on the source chain
-// and on the Mezo chain.
+// AssetsUnlockedEvent represents the event where inbound assets are released
+// from the bridge.
```

Comment:
I would also consider storing the `sender` just in case we need to implement bridge out timeouts one day. We need to know where to return funds.